mpact_api/telegram_bot/settings.py

```diff
@@ -0,0 +1,126 @@
+"""
+Django settings for telegram_bot project.
+
+Generated by 'django-admin startproject' using Django 2.2.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/2.2/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/2.2/ref/settings/
+"""
+
+import os
+
+# Build paths inside the project like this: os.path.join(BASE_DIR, ...)
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/2.2/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = os.environ.get("SECRET_KEY")
+
+# SECURITY WARNING: don't run with debug turned on in production!
+DEBUG = True
+
+ALLOWED_HOSTS = ["localhost", "8a7e5dd2d909.ngrok.io"]
```

Comment:
If you are using ngrok.io for dev, you could specify it in a `localsettings.py` file that you import at the end of `settings.py` to override some settings for your local environment, or set `ALLOWED_HOSTS` using an environment variable. e.g.
```suggestion
ALLOWED_HOSTS = os.environ['ALLOWED_HOSTS'].split(',')
```